Dockerfile

```diff
@@ -6,83 +6,58 @@ RUN /tmp/build-pgloader && rm /tmp/build-pgloader
 FROM ruby:2.6-stretch
 MAINTAINER operations@openproject.com
 
-ENV NODE_VERSION "10.15.0"
-ENV BUNDLER_VERSION "2.0.2"
+# Allow platform-specific additions. Valid values are: on-premise,cloud
+ARG PLATFORM=on-premise
+# Use OAuth token in case private gems need to be fetched
+ARG GITHUB_OAUTH_TOKEN
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV NODE_VERSION="12.18.3"
+ENV BUNDLER_VERSION="2.0.2"
 ENV BUNDLE_PATH__SYSTEM=false
-ENV APP_USER app
-ENV APP_PATH /app
-ENV APP_DATA_PATH /var/openproject/assets
-ENV APP_DATA_PATH_LEGACY /var/db/openproject
-ENV PGDATA /var/openproject/pgdata
-ENV PGDATA_LEGACY /var/lib/postgresql/9.6/main
-
-ENV DATABASE_URL postgres://openproject:openproject@127.0.0.1/openproject
-ENV RAILS_ENV production
-ENV HEROKU true
-ENV RAILS_CACHE_STORE memcache
-ENV OPENPROJECT_INSTALLATION__TYPE docker
-ENV NEW_RELIC_AGENT_ENABLED false
-ENV ATTACHMENTS_STORAGE_PATH $APP_DATA_PATH/files
+ENV APP_USER=app
+ENV APP_PATH=/app
+ENV APP_DATA_PATH=/var/openproject/assets
+ENV APP_DATA_PATH_LEGACY=/var/db/openproject
+ENV PGBIN="/usr/lib/postgresql/9.6/bin"
+ENV PGDATA=/var/openproject/pgdata
+ENV PGDATA_LEGACY=/var/lib/postgresql/9.6/main
+
+ENV DATABASE_URL=postgres://openproject:openproject@127.0.0.1/openproject
+ENV HEROKU=true
+ENV RAILS_ENV=production
+ENV RAILS_CACHE_STORE=memcache
+ENV BUNDLER_GROUPS="production docker opf_plugins"
+ENV OPENPROJECT_INSTALLATION__TYPE=docker
+# Valid values are: standard,bim
+ENV OPENPROJECT_EDITION=standard
+ENV NEW_RELIC_AGENT_ENABLED=false
+ENV ATTACHMENTS_STORAGE_PATH=$APP_DATA_PATH/files
 # Set a default key base, ensure to provide a secure value in production environments!
-ENV SECRET_KEY_BASE OVERWRITE_ME
+ENV SECRET_KEY_BASE=OVERWRITE_ME
 
 COPY --from=pgloader /usr/local/bin/pgloader-ccl /usr/local/bin/
```

Comment:
@oliverguenther @machisuji do you have an idea when we might remove pgloader from the docker image? How long has it been since the switch from MySQL?

Reply:
As discussed, I would vote to leave it in until we remove the respective migration tools. If it's taking too much space, that is